apps/twap/src/app/plugins/env.ts

```diff
@@ -26,6 +30,18 @@ const schema = {
     DATABASE_PASSWORD: {
       type: 'string',
     },
+    ORDERBOOK_DATABASE_HOST: {
```

Comment:
If there's 2 databases, shou't both have a prefix name? I mean, if `ORDERBOOK_DATABASE_HOST` shouldn't we name `DATABASE_NAME` --> `XXXXX_DATABASE_NAME`?